zeeguu/core/util/scroll_last_reading_percentage.py

```diff
@@ -0,0 +1,38 @@
+def find_last_reading_percentage(list, max_jump=10, max_total_update=40):
```

Comment:
we could almost create a `user_behavior` or `behavioral_estimation`, or `behavioral_modeling` module or something similar to put this there? it's more central to the app than just a `util`.